client/reader/update-notice/index.jsx

```diff
@@ -60,4 +49,8 @@ var UpdateNotice = React.createClass( {
 	}
 } );
 
-module.exports = UpdateNotice;
+export default connect(
```

Comment:
I'm unsure if it was this pull request I commented on already, but keep in mind `connect`ed components are pure by default, so double-check the other props to see if they're the type to get in trouble (objects, arrays, or functions expected to be mutated).